Formula/rdkit.rb

```diff
@@ -0,0 +1,80 @@
+class Rdkit < Formula
+  desc "Interpreted, interactive, object-oriented programming language"
+  homepage "https://rdkit.org/"
+  url "https://github.com/rdkit/rdkit/archive/Release_2020_09_4.tar.gz"
+  sha256 "9e734ca8f99d8be1ef2ac51efb67c393c62e88b98cfa550d6173ce3eaa87b559"
+  license "BSD-3-Clause"
+  head "https://github.com/rdkit/rdkit.git"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "boost"
+  depends_on "boost-python3"
+  depends_on "eigen"
+  depends_on "freetype"
+  depends_on "numpy"
+  depends_on "postgresql@12"
+  depends_on "py3cairo"
+  depends_on "python@3.9"
+
+  def install
+    ENV.cxx11
+    ENV.libcxx
+    ENV.append "CFLAGS", "-Wno-parentheses -Wno-logical-op-parentheses -Wno-format"
+    ENV.append "CXXFLAGS", "-Wno-parentheses -Wno-logical-op-parentheses -Wno-format"
+
+    # Get Python location
+    python_executable = Formula["python@3.9"].opt_bin/"python3"
+    py3ver = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+    py3prefix = Formula["python@3.9"].opt_frameworks/"Python.framework/Versions/#{py3ver}"
+    py3include = "#{py3prefix}/include/python#{py3ver}"
+
+    # set -DMAEPARSER and COORDGEN_FORCE_BUILD=ON to avoid conflicts with some formulae i.e. open-babel
+    args = std_cmake_args + %W[
+      -DRDK_INSTALL_INTREE=OFF
+      -DRDK_BUILD_SWIG_WRAPPERS=OFF
+      -DRDK_BUILD_AVALON_SUPPORT=ON
+      -DRDK_BUILD_PGSQL=ON
+      -DRDK_PGSQL_STATIC=ON
+      -DMAEPARSER_FORCE_BUILD=ON
+      -DCOORDGEN_FORCE_BUILD=ON
+      -DRDK_BUILD_INCHI_SUPPORT=ON
+      -DRDK_BUILD_CPP_TESTS=OFF
+      -DRDK_INSTALL_STATIC_LIBS=OFF
+      -DRDK_BUILD_CAIRO_SUPPORT=ON
+      -DRDK_BUILD_YAEHMOP_SUPPORT=ON
+      -DRDK_BUILD_FREESASA_SUPPORT=ON
+      -DBoost_NO_BOOST_CMAKE=ON
+      -DPYTHON_INCLUDE_DIR=#{py3include}
+      -DPYTHON_EXECUTABLE=#{python_executable}
+    ]
+
+    system "cmake", ".", *args
+    system "make"
+    system "make", "install"
+
+    # Remove the ghost .cmake files which will cause a warning if we install them to "lib"
+    rm_f Dir["#{lib}/*.cmake"]
+
+    # '3.8' is the default_version in Code/PgSQL/rdkit/rdkit.control
+    # This is a workaround to reproduce their filenames after installation.
+    (share/"postgresql/extension").install "Code/PgSQL/rdkit/rdkit--3.8.sql"
+    (share/"postgresql/extension").install "Code/PgSQL/rdkit/rdkit--3.8.control"
+    (lib/"postgresql").install "Code/PgSQL/rdkit/rdkit.so"
```

Comment:
This needs to be submitted and accepted as a patch upstream. We don't want manual patching in new formulae.

Reply:
OK, I will remove them and retry this PR because this workaround may no longer be required.